charts/icm/templates/test-job.yaml

```diff
@@ -84,7 +84,7 @@ spec:
         - -c
         - |
         {{- if index .Values "icm-as" "replication" "enabled" }}
-          until wget -qO- http://{{ .Release.Name }}-icm-as-live:7743/status/ReadinessProbe | grep -o "RUNNING" && wget -qO- http://{{ .Release.Name }}-icm-as-edit:7743/status/ReadinessProbe | grep -o "RUNNING"; do
+          until wget -qO- http://{{ .Release.Name }}-icm-as-live:{{ index .Values "icm-as" "ports" "svc" }}/status/ReadinessProbe | grep -o "RUNNING" && wget -qO- http://{{ .Release.Name }}-icm-as-edit:{{ index .Values "icm-as" "ports" "svc" }}/status/ReadinessProbe | grep -o "RUNNING"; do
         {{- else }}
           until result=$(wget -qO- http://{{ .Release.Name }}-icm-as:7743/status/ReadinessProbe | grep -o "RUNNING"); do
```

Comment:
May use a parameterized port here too

Reply:
Yeah, I also thought so but I always fails with a NilPointer.

Reply:
`helm unittest --output-file unit.xml --output-type JUnit charts/icm`:
```
 FAIL  test with default values charts/icm/tests/default-values-test_test.yaml
        - testrunner-job is correct
                Error: template: icm/templates/test-job.yaml:89:72: executing "icm/templates/test-job.yaml" at <index .Values "icm-as" "ports" "svc">: error calling index: index of nil pointer
```

Reply:
Please use management port for readiness probe.

Reply:
Please do not make ports configurable via values.yaml - we only need them as references.

Reply:
The unit test does also work now.